Sources/client-sdk-swift/momento/TopicClient.swift

```diff
@@ -11,19 +15,52 @@ public class TopicClient: TopicClientProtocol {
     var credentialProvider: CredentialProviderProtocol
     
     @available(macOS 10.15, *)
-    init(configuration: TopicClientConfiguration, credentialProvider: CredentialProviderProtocol, logger: MomentoLoggerProtocol = DefaultMomentoLogger(loggerName: "swift-momento-logger", level: DefaultMomentoLoggerLevel.info)) {
+    init(
+        configuration: TopicClientConfiguration,
+        credentialProvider: CredentialProviderProtocol,
+        logger: MomentoLoggerProtocol = DefaultMomentoLogger(
+            loggerName: "swift-momento-logger",
+            level: DefaultMomentoLoggerLevel.info
+        )
+    ) {
         self.logger = logger
         self.credentialProvider = credentialProvider
-        self.pubsubClient = PubsubClient(logger: logger, configuration: configuration, credentialProvider: credentialProvider)
+        self.pubsubClient = PubsubClient(
+            logger: logger,
+            configuration: configuration,
+            credentialProvider: credentialProvider
+        )
     }
     
-    public func publish() async -> String {
+    public func publish(
+        cacheName: String,
+        topicName: String,
+        value: String
+    ) async -> TopicPublishResponse {
+        if cacheName.count < 1 {
```

Comment:
Swift has an `isEmpty` string property that may be more idiomatic for this, but both that property and this solution will consider "  " valid. I found a neat [extension method](https://useyourloaf.com/blog/empty-strings-in-swift/) implementing `isBlank` we could use, but I think maybe we should hold off on adding and using the extension method until we have [validation factored out](https://github.com/momentohq/client-sdk-swift/issues/21) into it's own class/namespace. This is fine for now but let's revisit when we handle that ticket.